docs/source/conf.py

```diff
@@ -17,15 +17,15 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'ndx-nirs'
-copyright = '2021, Sumner Norman,Darin Erat Sleiter,José Ribeiro'
-author = 'Sumner Norman,Darin Erat Sleiter,José Ribeiro'
+project = "ndx-nirs"
+copyright = "2021, Sumner Norman,Darin Erat Sleiter,José Ribeiro"
+author = "Sumner Norman,Darin Erat Sleiter,José Ribeiro"
```

Comment:
This change was from running `black`. TBH I have a slight personal preference `'` except for docstrings (where double quotes are preferred by PEP8). However, I prefer following a single convention via `black` rather than worrying about individual convention preferences, and `black` uses `"`.

I usually end up writing with single quotes anyways since it doesn't require the use of `shift` and then let `black` do the conversion before I commit.